<!DOCTYPE html>
<html lang="en">
<head>
  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
</head>

<table class="table table-bordered table-hover table-condensed">
<thead><tr class="active"><th title="Field #1">ascot_website_id</th>
<th title="Field #2">citation</th>
<th title="Field #3">title</th>
<th title="Field #4">sra_project_id</th>
<th title="Field #5">pubmed_id</th>
<th title="Field #6">pubmed_link</th>
<th title="Field #7">full_description</th>
</tr></thead>
<tbody><tr>
<td>Brain</td>
<td>Barbosa-Morais et al. Science 2012</td>
<td>The evolutionary landscape of alternative splicing in vertebrate species</td>
<td>SRP015997</td>
<td align="right">23258890</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/23258890</td>
<td>Brain</td>
</tr>
<tr>
<td>Brain</td>
<td>Merkin et al. Science 2012</td>
<td>Evolutionary dynamics of gene and isoform regulation in Mammalian tissues</td>
<td>SRP016501</td>
<td align="right">23258891</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/23258891</td>
<td>Brain</td>
</tr>
<tr>
<td>Skeletal Muscle</td>
<td>Barbosa-Morais et al. Science 2012</td>
<td>The evolutionary landscape of alternative splicing in vertebrate species</td>
<td>SRP015997</td>
<td align="right">23258890</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/23258890</td>
<td>Skeletal Muscle</td>
</tr>
<tr>
<td>Skeletal Muscle</td>
<td>Merkin et al. Science 2012</td>
<td>Evolutionary dynamics of gene and isoform regulation in Mammalian tissues</td>
<td>SRP016501</td>
<td align="right">23258891</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/23258891</td>
<td>Skeletal Muscle</td>
</tr>
<tr>
<td>Heart</td>
<td>Barbosa-Morais et al. Science 2012</td>
<td>The evolutionary landscape of alternative splicing in vertebrate species</td>
<td>SRP015997</td>
<td align="right">23258890</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/23258890</td>
<td>Heart</td>
</tr>
<tr>
<td>Lung</td>
<td>Merkin et al. Science 2012</td>
<td>Evolutionary dynamics of gene and isoform regulation in Mammalian tissues</td>
<td>SRP016501</td>
<td align="right">23258891</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/23258891</td>
<td>Lung</td>
</tr>
<tr>
<td>Stomach</td>
<td>Tan et al. Nature 2017</td>
<td>Dynamic landscape and regulation of RNA editing in mammals</td>
<td>SRP039090</td>
<td align="right">29022589</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/29022589</td>
<td>Stomach</td>
</tr>
<tr>
<td>Spleen</td>
<td>Merkin et al. Science 2012</td>
<td>Evolutionary dynamics of gene and isoform regulation in Mammalian tissues</td>
<td>SRP016501</td>
<td align="right">23258891</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/23258891</td>
<td>Spleen</td>
</tr>
<tr>
<td>Colon</td>
<td>Merkin et al. Science 2012</td>
<td>Evolutionary dynamics of gene and isoform regulation in Mammalian tissues</td>
<td>SRP016501</td>
<td align="right">23258891</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/23258891</td>
<td>Colon</td>
</tr>
<tr>
<td>Liver</td>
<td>Ding et al. Mol Cell Proteomics. 2016</td>
<td>A Cell-type-resolved Liver Proteome</td>
<td>SRP033468</td>
<td align="right">27562671</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/27562671</td>
<td>Liver</td>
</tr>
<tr>
<td>Liver</td>
<td>Barbosa-Morais et al. Science 2012</td>
<td>The evolutionary landscape of alternative splicing in vertebrate species</td>
<td>SRP015997</td>
<td align="right">23258890</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/23258890</td>
<td>Liver</td>
</tr>
<tr>
<td>Liver</td>
<td>Merkin et al. Science 2012</td>
<td>Evolutionary dynamics of gene and isoform regulation in Mammalian tissues</td>
<td>SRP016501</td>
<td align="right">23258891</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/23258891</td>
<td>Liver</td>
</tr>
<tr>
<td>Kidney</td>
<td>Barbosa-Morais et al. Science 2012</td>
<td>The evolutionary landscape of alternative splicing in vertebrate species</td>
<td>SRP015997</td>
<td align="right">23258890</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/23258890</td>
<td>Kidney</td>
</tr>
<tr>
<td>Kidney</td>
<td>Merkin et al. Science 2012</td>
<td>Evolutionary dynamics of gene and isoform regulation in Mammalian tissues</td>
<td>SRP016501</td>
<td align="right">23258891</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/23258891</td>
<td>Kidney</td>
</tr>
<tr>
<td>Pancreas islet</td>
<td>Perelis et al. Science 2015</td>
<td>Pancreatic Î² cell enhancers regulate rhythmic transcription of genes controlling insulin secretion</td>
<td>SRP059509</td>
<td align="right">26542580</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/26542580</td>
<td>Pancreas islet</td>
</tr>
<tr>
<td>Mammary gland</td>
<td>Goel et al. Cancer Cell 2016</td>
<td>Overcoming Therapeutic Resistance in HER2-Positive Breast Cancers with CDK4/6 Inhibitors</td>
<td>SRP060340</td>
<td align="right">26977878</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/26977878</td>
<td>Mammary gland</td>
</tr>
<tr>
<td>Ovary</td>
<td>Kabayama et al. Nucleic Acids Res. 2017</td>
<td>Roles of MIWI, MILI and PLD6 in small RNA regulation in mouse growing oocytes</td>
<td>DRP003418</td>
<td align="right">28115634</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/28115634</td>
<td>Ovary</td>
</tr>
<tr>
<td>Testes</td>
<td>Merkin et al. Science 2012</td>
<td>Evolutionary dynamics of gene and isoform regulation in Mammalian tissues</td>
<td>SRP016501</td>
<td align="right">23258891</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/23258891</td>
<td>Testes</td>
</tr>
<tr>
<td>(p2) Rod Photoreceptor</td>
<td>Kim et al. Dev Cell. 2016</td>
<td>Recruitment of Rod Photoreceptors from Short-Wavelength-Sensitive Cones during the Evolution of Nocturnal Vision in Mammals</td>
<td>SRP065767</td>
<td align="right">27326930</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/27326930</td>
<td>Rod Photoreceptor (p2)</td>
</tr>
<tr>
<td>(p4) Rod Photoreceptor</td>
<td>Kim et al. Dev Cell. 2016</td>
<td>Recruitment of Rod Photoreceptors from Short-Wavelength-Sensitive Cones during the Evolution of Nocturnal Vision in Mammals</td>
<td>SRP065767</td>
<td align="right">27326930</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/27326930</td>
<td>Rod Photoreceptor (p4)</td>
</tr>
<tr>
<td>(p6) Rod Photoreceptor</td>
<td>Kim et al. Dev Cell. 2016</td>
<td>Recruitment of Rod Photoreceptors from Short-Wavelength-Sensitive Cones during the Evolution of Nocturnal Vision in Mammals</td>
<td>SRP065767</td>
<td align="right">27326930</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/27326930</td>
<td>Rod Photoreceptor (p6)</td>
</tr>
<tr>
<td>(p10) Rod Photoreceptor</td>
<td>Kim et al. Dev Cell. 2016</td>
<td>Recruitment of Rod Photoreceptors from Short-Wavelength-Sensitive Cones during the Evolution of Nocturnal Vision in Mammals</td>
<td>SRP065767</td>
<td align="right">27326930</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/27326930</td>
<td>Rod Photoreceptor (p10)</td>
</tr>
<tr>
<td>(p14) Rod Photoreceptor</td>
<td>Kim et al. Dev Cell. 2016</td>
<td>Recruitment of Rod Photoreceptors from Short-Wavelength-Sensitive Cones during the Evolution of Nocturnal Vision in Mammals</td>
<td>SRP065767</td>
<td align="right">27326930</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/27326930</td>
<td>Rod Photoreceptor (p14)</td>
</tr>
<tr>
<td>(p28) Rod Photoreceptor</td>
<td>Kim et al. Dev Cell. 2016</td>
<td>Recruitment of Rod Photoreceptors from Short-Wavelength-Sensitive Cones during the Evolution of Nocturnal Vision in Mammals</td>
<td>SRP065767</td>
<td align="right">27326930</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/27326930</td>
<td>Rod Photoreceptor (p28)</td>
</tr>
<tr>
<td>(cortex) SST interneurons</td>
<td>Yang et al. BMC Genomics 2017</td>
<td>Chronic nicotine differentially affects murine transcriptome profiling in isolated cortical interneurons and pyramidal neurons</td>
<td>SRP096044</td>
<td align="right">28219337</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/28219337</td>
<td>SST interneurons (cortex)</td>
</tr>
<tr>
<td>(hippocampus) PV interneurons</td>
<td>Cembrowski et al. Neuron 2016</td>
<td>Spatial Gene-Expression Gradients Underlie Prominent Heterogeneity of CA1 Pyramidal Neurons</td>
<td>SRP056666</td>
<td align="right">26777276</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/26777276</td>
<td>PV interneurons (hippocampus)</td>
</tr>
<tr>
<td>Embryonic motor neurons</td>
<td>Amin et al. Science 2015</td>
<td>Loss of motoneuron-specific microRNA-218 causes systemic neuromuscular failure</td>
<td>SRP066961</td>
<td align="right">26680198</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/26680198</td>
<td>Motor neurons (Hb9+, embryo)</td>
</tr>
<tr>
<td>DRG neurons</td>
<td>Tedeschi et al. Neuron 2016</td>
<td>The Calcium Channel Subunit Alpha2delta2 Suppresses Axon Regeneration in the Adult CNS</td>
<td>SRP055201</td>
<td align="right">27720483</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/27720483</td>
<td>DRG neurons</td>
</tr>
<tr>
<td>Olfactory sensory neurons</td>
<td>Saraiva et al. Sci Rep. 2015</td>
<td>Hierarchical deconstruction of mouse olfactory sensory neurons: from whole mucosa to single-cell RNA-seq</td>
<td>ERP009139</td>
<td align="right">26670777</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/26670777</td>
<td>Olfactory sensory neurons (Omp+)</td>
</tr>
<tr>
<td>Cochlear hair cells</td>
<td>Cai et al. J Neurosci. 2015</td>
<td>Characterization of the transcriptome of nascent hair cells and identification of direct targets of the Atoh1 transcription factor</td>
<td>SRP053189</td>
<td align="right">25855195</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/25855195</td>
<td>Hair cells (Atoh1+, cochlea)</td>
</tr>
<tr>
<td>(cerebellum) Purkinje cells</td>
<td>Tan et al. Hum Mol Genet. 2016</td>
<td>Extensive cryptic splicing upon loss of RBM17 and TDP43 in neurodegeneration models</td>
<td>SRP071321</td>
<td align="right">28007900</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/28007900</td>
<td>Purkinje cells (cerebellum)</td>
</tr>
<tr>
<td>(cortex) Pyramidal neurons</td>
<td>Yang et al. BMC Genomics 2017</td>
<td>Chronic nicotine differentially affects murine transcriptome profiling in isolated cortical interneurons and pyramidal neurons</td>
<td>SRP096044</td>
<td align="right">28219337</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/28219337</td>
<td>Pyramidal neurons (Thy1+, cortex)</td>
</tr>
<tr>
<td>Proximal CA1 - Pyramidal</td>
<td>Cembrowski et al. Neuron 2016</td>
<td>Spatial Gene-Expression Gradients Underlie Prominent Heterogeneity of CA1 Pyramidal Neurons</td>
<td>SRP056666</td>
<td align="right">26777276</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/26777276</td>
<td>Pyramidal neurons (hippocampus - proximal CA1)</td>
</tr>
<tr>
<td>Distal CA1 - Pyramidal</td>
<td>Cembrowski et al. Neuron 2016</td>
<td>Spatial Gene-Expression Gradients Underlie Prominent Heterogeneity of CA1 Pyramidal Neurons</td>
<td>SRP056666</td>
<td align="right">26777276</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/26777276</td>
<td>Pyramidal neurons (hippocampus - distal CA1)</td>
</tr>
<tr>
<td>Superficial CA1 - Pyramidal</td>
<td>Cembrowski et al. Neuron 2016</td>
<td>Spatial Gene-Expression Gradients Underlie Prominent Heterogeneity of CA1 Pyramidal Neurons</td>
<td>SRP056666</td>
<td align="right">26777276</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/26777276</td>
<td>Pyramidal neurons (hippocampus - superficial CA1)</td>
</tr>
<tr>
<td>Dorsal CA1 - Pyramidal</td>
<td>Cembrowski et al. Neuron 2016</td>
<td>Spatial Gene-Expression Gradients Underlie Prominent Heterogeneity of CA1 Pyramidal Neurons</td>
<td>SRP056666</td>
<td align="right">26777276</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/26777276</td>
<td>Pyramidal neurons (hippocampus - dorsal CA1)</td>
</tr>
<tr>
<td>Dorsal CA1 â PoS - Pyramidal</td>
<td>Cembrowski et al. Neuron 2016</td>
<td>Spatial Gene-Expression Gradients Underlie Prominent Heterogeneity of CA1 Pyramidal Neurons</td>
<td>SRP056666</td>
<td align="right">26777276</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/26777276</td>
<td>Pyramidal neurons (hippocampus - dorsal CA1, postsubiculum-projecting)</td>
</tr>
<tr>
<td>Intermediate CA1 - Pyramidal</td>
<td>Cembrowski et al. Neuron 2016</td>
<td>Spatial Gene-Expression Gradients Underlie Prominent Heterogeneity of CA1 Pyramidal Neurons</td>
<td>SRP056666</td>
<td align="right">26777276</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/26777276</td>
<td>Pyramidal neurons (hippocampus - intermediate CA1)</td>
</tr>
<tr>
<td>Ventral CA1 - Pyramidal</td>
<td>Cembrowski et al. Neuron 2016</td>
<td>Spatial Gene-Expression Gradients Underlie Prominent Heterogeneity of CA1 Pyramidal Neurons</td>
<td>SRP056666</td>
<td align="right">26777276</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/26777276</td>
<td>Pyramidal neurons (hippocampus - ventral CA1)</td>
</tr>
<tr>
<td>Ventral CA1 â Amy - Pyramidal</td>
<td>Cembrowski et al. Neuron 2016</td>
<td>Spatial Gene-Expression Gradients Underlie Prominent Heterogeneity of CA1 Pyramidal Neurons</td>
<td>SRP056666</td>
<td align="right">26777276</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/26777276</td>
<td>Pyramidal neurons (hippocampus - ventral CA1, amygdala-projecting)</td>
</tr>
<tr>
<td>Ventral CA1 â NAc - Pyramidal</td>
<td>Cembrowski et al. Neuron 2016</td>
<td>Spatial Gene-Expression Gradients Underlie Prominent Heterogeneity of CA1 Pyramidal Neurons</td>
<td>SRP056666</td>
<td align="right">26777276</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/26777276</td>
<td>Pyramidal neurons (hippocampus - ventral CA1, nucleus accumbens-projecting)</td>
</tr>
<tr>
<td>Deep CA1 - Pyramidal</td>
<td>Cembrowski et al. Neuron 2016</td>
<td>Spatial Gene-Expression Gradients Underlie Prominent Heterogeneity of CA1 Pyramidal Neurons</td>
<td>SRP056666</td>
<td align="right">26777276</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/26777276</td>
<td>Pyramidal neurons (hippocampus - deep CA1)</td>
</tr>
<tr>
<td>Dorsal CA3 - Pyramidal</td>
<td>Cembrowski et al. Neuron 2016</td>
<td>Spatial Gene-Expression Gradients Underlie Prominent Heterogeneity of CA1 Pyramidal Neurons</td>
<td>SRP056666</td>
<td align="right">26777276</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/26777276</td>
<td>Pyramidal neurons (hippocampus - dorsal CA3)</td>
</tr>
<tr>
<td>Ventral CA3 - Pyramidal</td>
<td>Cembrowski et al. Neuron 2016</td>
<td>Spatial Gene-Expression Gradients Underlie Prominent Heterogeneity of CA1 Pyramidal Neurons</td>
<td>SRP056666</td>
<td align="right">26777276</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/26777276</td>
<td>Pyramidal neurons (hippocampus - ventral CA3)</td>
</tr>
<tr>
<td>Dorsal DG - Granule</td>
<td>Cembrowski et al. Neuron 2016</td>
<td>Spatial Gene-Expression Gradients Underlie Prominent Heterogeneity of CA1 Pyramidal Neurons</td>
<td>SRP056666</td>
<td align="right">26777276</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/26777276</td>
<td>Granule cells (hippocampus - dorsal DG)</td>
</tr>
<tr>
<td>Ventral DG - Granule</td>
<td>Cembrowski et al. Neuron 2016</td>
<td>Spatial Gene-Expression Gradients Underlie Prominent Heterogeneity of CA1 Pyramidal Neurons</td>
<td>SRP056666</td>
<td align="right">26777276</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/26777276</td>
<td>Granule cells (hippocampus - ventral DG)</td>
</tr>
<tr>
<td>(cortex) Oligo precursor cells</td>
<td>Zhang et al. J Neurosci. 2014</td>
<td>An RNA-sequencing transcriptome and splicing database of glia, neurons, and vascular cells of the cerebral cortex</td>
<td>SRP033200</td>
<td align="right">25186741</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/25186741</td>
<td>Oligodendrocyte precursor cells (cortex)</td>
</tr>
<tr>
<td>(cortex) Newly formed oligos</td>
<td>Zhang et al. J Neurosci. 2014</td>
<td>An RNA-sequencing transcriptome and splicing database of glia, neurons, and vascular cells of the cerebral cortex</td>
<td>SRP033200</td>
<td align="right">25186741</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/25186741</td>
<td>Newly formed oligodendrocytes (cortex)</td>
</tr>
<tr>
<td>(cortex) Myelinating oligos</td>
<td>Zhang et al. J Neurosci. 2014</td>
<td>An RNA-sequencing transcriptome and splicing database of glia, neurons, and vascular cells of the cerebral cortex</td>
<td>SRP033200</td>
<td align="right">25186741</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/25186741</td>
<td>Myelinating oligodendrocytes (cortex)</td>
</tr>
<tr>
<td>(cortex) Astrocytes</td>
<td>Zhang et al. J Neurosci. 2014</td>
<td>An RNA-sequencing transcriptome and splicing database of glia, neurons, and vascular cells of the cerebral cortex</td>
<td>SRP033200</td>
<td align="right">25186741</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/25186741</td>
<td>Astrocytes (cortex)</td>
</tr>
<tr>
<td>(cortex) Microglia</td>
<td>Zhang et al. J Neurosci. 2014</td>
<td>An RNA-sequencing transcriptome and splicing database of glia, neurons, and vascular cells of the cerebral cortex</td>
<td>SRP033200</td>
<td align="right">25186741</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/25186741</td>
<td>Microglia (cortex)</td>
</tr>
<tr>
<td>(spinal cord) Microglia</td>
<td>Denk et al. Cell Rep. 2016</td>
<td>Persistent Alterations in Microglial Enhancers in a Model of Chronic Pain</td>
<td>SRP061355</td>
<td align="right">27184839</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/27184839</td>
<td>Microglia (spinal cord)</td>
</tr>
<tr>
<td>(cortex) Endothelial cells</td>
<td>Zhang et al. J Neurosci. 2014</td>
<td>An RNA-sequencing transcriptome and splicing database of glia, neurons, and vascular cells of the cerebral cortex</td>
<td>SRP033200</td>
<td align="right">25186741</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/25186741</td>
<td>Endothelial cells (cortex)</td>
</tr>
<tr>
<td>(Pax7+) Muscle stem cells</td>
<td>Liu et al. Nat Cell Biol. 2017</td>
<td>A Twist2-dependent progenitor cell contributes to adult skeletal muscle</td>
<td>SRP078457</td>
<td align="right">28218909</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/28218909</td>
<td>Muscle stem cells (Pax7+)</td>
</tr>
<tr>
<td>(Tw2+) Muscle stem cells</td>
<td>Liu et al. Nat Cell Biol. 2017</td>
<td>A Twist2-dependent progenitor cell contributes to adult skeletal muscle</td>
<td>SRP078457</td>
<td align="right">28218909</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/28218909</td>
<td>Muscle stem cells (Tw2+)</td>
</tr>
<tr>
<td>Quadriceps</td>
<td>Zygmunt et al. Mol Cell Biol. 2017</td>
<td>Deletion of Pofut1 in Mouse Skeletal Myofibers Induces Muscle Aging-Related Phenotypes in cis and in trans</td>
<td>SRP099194</td>
<td align="right">28265002</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/28265002</td>
<td>Quadriceps</td>
</tr>
<tr>
<td>Gastrocnemius</td>
<td>Liu et al. Elife 2017</td>
<td>Loss of adult skeletal muscle stem cells drives age-related neuromuscular junction degeneration</td>
<td>SRP107891</td>
<td align="right">28583253</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/28583253</td>
<td>Gastrocnemius</td>
</tr>
<tr>
<td>(extrasynaptic) Diaphragm</td>
<td>Liu et al. Elife 2017</td>
<td>Loss of adult skeletal muscle stem cells drives age-related neuromuscular junction degeneration</td>
<td>SRP107891</td>
<td align="right">28583253</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/28583253</td>
<td>Diaphragm, extrasynaptic</td>
</tr>
<tr>
<td>(synaptic) Diaphragm</td>
<td>Liu et al. Elife 2017</td>
<td>Loss of adult skeletal muscle stem cells drives age-related neuromuscular junction degeneration</td>
<td>SRP107891</td>
<td align="right">28583253</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/28583253</td>
<td>Diaphragm, synaptic</td>
</tr>
<tr>
<td>(neonatal) Cardiomyocytes</td>
<td>Giudice et al. Nat Commun. 2014</td>
<td>Alternative splicing regulates vesicular trafficking genes in cardiomyocytes during postnatal heart development</td>
<td>SRP029464</td>
<td align="right">24752171</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/24752171</td>
<td>Cardiomyocytes (neonatal)</td>
</tr>
<tr>
<td>(adult) Cardiomyocytes</td>
<td>Giudice et al. Nat Commun. 2014</td>
<td>Alternative splicing regulates vesicular trafficking genes in cardiomyocytes during postnatal heart development</td>
<td>SRP029464</td>
<td align="right">24752171</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/24752171</td>
<td>Cardiomyocytes (adult)</td>
</tr>
<tr>
<td>(neonatal) Cardiac fibroblasts</td>
<td>Giudice et al. Nat Commun. 2014</td>
<td>Alternative splicing regulates vesicular trafficking genes in cardiomyocytes during postnatal heart development</td>
<td>SRP029464</td>
<td align="right">24752171</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/24752171</td>
<td>Cardiac fibroblasts (neonatal)</td>
</tr>
<tr>
<td>(adult) Cardiac fibroblasts</td>
<td>Giudice et al. Nat Commun. 2014</td>
<td>Alternative splicing regulates vesicular trafficking genes in cardiomyocytes during postnatal heart development</td>
<td>SRP029464</td>
<td align="right">24752171</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/24752171</td>
<td>Cardiac fibroblasts (adult)</td>
</tr>
<tr>
<td>(e17) Heart ventricle</td>
<td>Giudice et al. Nat Commun. 2014</td>
<td>Alternative splicing regulates vesicular trafficking genes in cardiomyocytes during postnatal heart development</td>
<td>SRP029464</td>
<td align="right">24752171</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/24752171</td>
<td>Heart ventricle (e17)</td>
</tr>
<tr>
<td>(p1) Heart ventricle</td>
<td>Giudice et al. Nat Commun. 2014</td>
<td>Alternative splicing regulates vesicular trafficking genes in cardiomyocytes during postnatal heart development</td>
<td>SRP029464</td>
<td align="right">24752171</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/24752171</td>
<td>Heart ventricle (p1)</td>
</tr>
<tr>
<td>(p10) Heart ventricle</td>
<td>Giudice et al. Nat Commun. 2014</td>
<td>Alternative splicing regulates vesicular trafficking genes in cardiomyocytes during postnatal heart development</td>
<td>SRP029464</td>
<td align="right">24752171</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/24752171</td>
<td>Heart ventricle (p10)</td>
</tr>
<tr>
<td>(p28) Heart ventricle</td>
<td>Giudice et al. Nat Commun. 2014</td>
<td>Alternative splicing regulates vesicular trafficking genes in cardiomyocytes during postnatal heart development</td>
<td>SRP029464</td>
<td align="right">24752171</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/24752171</td>
<td>Heart ventricle (p28)</td>
</tr>
<tr>
<td>(p90) Heart ventricle</td>
<td>Giudice et al. Nat Commun. 2014</td>
<td>Alternative splicing regulates vesicular trafficking genes in cardiomyocytes during postnatal heart development</td>
<td>SRP029464</td>
<td align="right">24752171</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/24752171</td>
<td>Heart ventricle (p90)</td>
</tr>
<tr>
<td>(marrow) Hematopoietic stem cell</td>
<td>Manesia et al. Stem Cell Res. 2015</td>
<td>Highly proliferative primitive fetal liver hematopoietic stem cells are fueled by oxidative metabolic pathways</td>
<td>ERP013145</td>
<td align="right">26599326</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/26599326</td>
<td>Hematopoietic stem cell (bone marrow)</td>
</tr>
<tr>
<td>(fetal liver) Hematopoietic stem cell</td>
<td>Manesia et al. Stem Cell Res. 2015</td>
<td>Highly proliferative primitive fetal liver hematopoietic stem cells are fueled by oxidative metabolic pathways</td>
<td>ERP013145</td>
<td align="right">26599326</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/26599326</td>
<td>Hematopoietic stem cell (fetal liver)</td>
</tr>
<tr>
<td>(spleen) Dendritic cells</td>
<td>Shay and Kang. Trends Immunol. 2013</td>
<td>Immunological Genome Project and systems immunology</td>
<td>SRP057261</td>
<td align="right">23631936</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/23631936</td>
<td>Dendritic cells (spleen)</td>
</tr>
<tr>
<td>(bone marrow) Dendritic cells</td>
<td>Shalek et al. Nature 2013</td>
<td>Single-cell transcriptomics reveals bimodality in expression and splicing in immune cells</td>
<td>SRP015959</td>
<td align="right">23685454</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/23685454</td>
<td>Dendritic cells (bone marrow)</td>
</tr>
<tr>
<td>(+IL10, +LPS) Dendritic cells</td>
<td>Hutchins et al. Sci Rep. 2015</td>
<td>Genomic analysis of LPS-stimulated myeloid cells identifies a common pro-inflammatory response but divergent IL-10 anti-inflammatory responses</td>
<td>SRP038980</td>
<td align="right">25765318</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/25765318</td>
<td>Dendritic cells (+IL10, +LPS)</td>
</tr>
<tr>
<td>Neutrophil</td>
<td>Shay and Kang. Trends Immunol. 2013</td>
<td>Immunological Genome Project and systems immunology</td>
<td>SRP057261</td>
<td align="right">23631936</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/23631936</td>
<td>Neutrophil</td>
</tr>
<tr>
<td>(+IL10, +LPS) Neutrophil</td>
<td>Hutchins et al. Sci Rep. 2015</td>
<td>Genomic analysis of LPS-stimulated myeloid cells identifies a common pro-inflammatory response but divergent IL-10 anti-inflammatory responses</td>
<td>SRP038980</td>
<td align="right">25765318</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/25765318</td>
<td>Neutrophil (+IL10, +LPS)</td>
</tr>
<tr>
<td>Mast cell</td>
<td>Hutchins et al. Sci Rep. 2015</td>
<td>Genomic analysis of LPS-stimulated myeloid cells identifies a common pro-inflammatory response but divergent IL-10 anti-inflammatory responses</td>
<td>SRP038980</td>
<td align="right">25765318</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/25765318</td>
<td>Mast cell</td>
</tr>
<tr>
<td>(+IL10, +LPS) Mast cell</td>
<td>Hutchins et al. Sci Rep. 2015</td>
<td>Genomic analysis of LPS-stimulated myeloid cells identifies a common pro-inflammatory response but divergent IL-10 anti-inflammatory responses</td>
<td>SRP038980</td>
<td align="right">25765318</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/25765318</td>
<td>Mast cell (+IL10, +LPS)</td>
</tr>
<tr>
<td>NKT cell</td>
<td>Shay and Kang. Trends Immunol. 2013</td>
<td>Immunological Genome Project and systems immunology</td>
<td>SRP057261</td>
<td align="right">23631936</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/23631936</td>
<td>NKT cell</td>
</tr>
<tr>
<td>NK cell</td>
<td>Shay and Kang. Trends Immunol. 2013</td>
<td>Immunological Genome Project and systems immunology</td>
<td>SRP057261</td>
<td align="right">23631936</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/23631936</td>
<td>NK cell</td>
</tr>
<tr>
<td>Macrophage</td>
<td>Shay and Kang. Trends Immunol. 2013</td>
<td>Immunological Genome Project and systems immunology</td>
<td>SRP057261</td>
<td align="right">23631936</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/23631936</td>
<td>Macrophage</td>
</tr>
<tr>
<td>Eosinophils</td>
<td>Hutchins et al. Sci Rep. 2015</td>
<td>Genomic analysis of LPS-stimulated myeloid cells identifies a common pro-inflammatory response but divergent IL-10 anti-inflammatory responses</td>
<td>SRP038980</td>
<td align="right">25765318</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/25765318</td>
<td>Eosinophils</td>
</tr>
<tr>
<td>(+LPS) Eosinophils</td>
<td>Hutchins et al. Sci Rep. 2015</td>
<td>Genomic analysis of LPS-stimulated myeloid cells identifies a common pro-inflammatory response but divergent IL-10 anti-inflammatory responses</td>
<td>SRP038980</td>
<td align="right">25765318</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/25765318</td>
<td>Eosinophils (+LPS)</td>
</tr>
<tr>
<td>Peritoneal exudate cells</td>
<td>Hutchins et al. Sci Rep. 2015</td>
<td>Genomic analysis of LPS-stimulated myeloid cells identifies a common pro-inflammatory response but divergent IL-10 anti-inflammatory responses</td>
<td>SRP038980</td>
<td align="right">25765318</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/25765318</td>
<td>Peritoneal exudate cells, adherent</td>
</tr>
<tr>
<td>(+IL10, +LPS) Peritoneal exudate</td>
<td>Hutchins et al. Sci Rep. 2015</td>
<td>Genomic analysis of LPS-stimulated myeloid cells identifies a common pro-inflammatory response but divergent IL-10 anti-inflammatory responses</td>
<td>SRP038980</td>
<td align="right">25765318</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/25765318</td>
<td>Peritoneal exudate cells, adherent (+IL10, +LPS)</td>
</tr>
<tr>
<td>(spleen) B cell</td>
<td>Shay and Kang. Trends Immunol. 2013</td>
<td>Immunological Genome Project and systems immunology</td>
<td>SRP057261</td>
<td align="right">23631936</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/23631936</td>
<td>B cell, spleen</td>
</tr>
<tr>
<td>(peritoneal cavity) B cell</td>
<td>Shay and Kang. Trends Immunol. 2013</td>
<td>Immunological Genome Project and systems immunology</td>
<td>SRP057261</td>
<td align="right">23631936</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/23631936</td>
<td>B cell, peritoneal cavity lavage</td>
</tr>
<tr>
<td>(+LPS) B cell</td>
<td>Campos-Sanchez et al. Cell Rep. 2017</td>
<td>Wolf-Hirschhorn Syndrome Candidate 1 Is Necessary for Correct Hematopoietic and B Cell Development</td>
<td>SRP079936</td>
<td align="right">28538178</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/28538178</td>
<td>B cell (+LPS)</td>
</tr>
<tr>
<td>(Th17) CD4+ T-cell</td>
<td>Stubbington et al. Biol Direct. 2015</td>
<td>An atlas of mouse CD4(+) T cell transcriptomes</td>
<td>ERP005852</td>
<td align="right">25886751</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/25886751</td>
<td>CD4+ T-cell (Th17)</td>
</tr>
<tr>
<td>(Th1) CD4+ T-cell</td>
<td>Stubbington et al. Biol Direct. 2015</td>
<td>An atlas of mouse CD4(+) T cell transcriptomes</td>
<td>ERP005852</td>
<td align="right">25886751</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/25886751</td>
<td>CD4+ T-cell (Th1)</td>
</tr>
<tr>
<td>(Th2) CD4+ T-cell</td>
<td>Stubbington et al. Biol Direct. 2015</td>
<td>An atlas of mouse CD4(+) T cell transcriptomes</td>
<td>ERP005852</td>
<td align="right">25886751</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/25886751</td>
<td>CD4+ T-cell (Th2)</td>
</tr>
<tr>
<td>CD8+ T-cell</td>
<td>Shay and Kang. Trends Immunol. 2013</td>
<td>Immunological Genome Project and systems immunology</td>
<td>SRP057261</td>
<td align="right">23631936</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/23631936</td>
<td>CD8+ T-cell</td>
</tr>
<tr>
<td>CD4+/CD8+ double-pos. T-cell</td>
<td>Scourzic et al. Leukemia 2016</td>
<td>DNMT3A(R882H) mutant and Tet2 inactivation cooperate in the deregulation of DNA methylation control to induce lymphoid malignancies in mice</td>
<td>ERP013684</td>
<td align="right">26876596</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/26876596</td>
<td>CD4+/CD8+ double positive T-cell</td>
</tr>
<tr>
<td>Gamma delta T-cell</td>
<td>Shay and Kang. Trends Immunol. 2013</td>
<td>Immunological Genome Project and systems immunology</td>
<td>SRP057261</td>
<td align="right">23631936</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/23631936</td>
<td>Gamma delta T-cell</td>
</tr>
<tr>
<td>(Rag2-, spleen) Regulatory T-cell</td>
<td>Thiault et al. Nat Immunol. 2015</td>
<td>Peripheral regulatory T lymphocytes recirculating to the thymus suppress the development of their precursors</td>
<td>SRP056191</td>
<td align="right">25939024</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/25939024</td>
<td>Regulatory T-cell (Rag2-, spleen)</td>
</tr>
<tr>
<td>(Rag2+, spleen) Regulatory T-cell</td>
<td>Thiault et al. Nat Immunol. 2015</td>
<td>Peripheral regulatory T lymphocytes recirculating to the thymus suppress the development of their precursors</td>
<td>SRP056191</td>
<td align="right">25939024</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/25939024</td>
<td>Regulatory T-cell (Rag2+, spleen)</td>
</tr>
<tr>
<td>(Rag2-, thymus) Regulatory T-cell</td>
<td>Thiault et al. Nat Immunol. 2015</td>
<td>Peripheral regulatory T lymphocytes recirculating to the thymus suppress the development of their precursors</td>
<td>SRP056191</td>
<td align="right">25939024</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/25939024</td>
<td>Regulatory T-cell (Rag2-, thymus)</td>
</tr>
<tr>
<td>(Rag2+, thymus) Regulatory T-cell</td>
<td>Thiault et al. Nat Immunol. 2015</td>
<td>Peripheral regulatory T lymphocytes recirculating to the thymus suppress the development of their precursors</td>
<td>SRP056191</td>
<td align="right">25939024</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/25939024</td>
<td>Regulatory T-cell (Rag2+, thymus)</td>
</tr>
<tr>
<td>(liver) Hepatocyte</td>
<td>Ding et al. Mol Cell Proteomics. 2016</td>
<td>A Cell-type-resolved Liver Proteome</td>
<td>SRP033468</td>
<td align="right">27562671</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/27562671</td>
<td>Hepatocyte (liver)</td>
</tr>
<tr>
<td>(liver) Sinusoidal endothelial cells</td>
<td>Ding et al. Mol Cell Proteomics. 2016</td>
<td>A Cell-type-resolved Liver Proteome</td>
<td>SRP033468</td>
<td align="right">27562671</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/27562671</td>
<td>Liver sinusoidal endothelial cells (liver)</td>
</tr>
<tr>
<td>(liver) Kupffer cells</td>
<td>Ding et al. Mol Cell Proteomics. 2016</td>
<td>A Cell-type-resolved Liver Proteome</td>
<td>SRP033468</td>
<td align="right">27562671</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/27562671</td>
<td>Kupffer cells (liver)</td>
</tr>
<tr>
<td>(liver) Hepatic stellate cells</td>
<td>Ding et al. Mol Cell Proteomics. 2016</td>
<td>A Cell-type-resolved Liver Proteome</td>
<td>SRP033468</td>
<td align="right">27562671</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/27562671</td>
<td>Hepatic stellate cells (liver)</td>
</tr>
<tr>
<td>(jejunum) Smooth muscle cells</td>
<td>Lee et al. PLoS One 2015</td>
<td>Smooth Muscle Cell Genome Browser: Enabling the Identification of Novel Serum Response Factor Target Genes</td>
<td>SRP042078</td>
<td align="right">26241044</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/26241044</td>
<td>Smooth muscle cells (jejunum)</td>
</tr>
<tr>
<td>(colon) Smooth muscle cells</td>
<td>Lee et al. PLoS One 2015</td>
<td>Smooth Muscle Cell Genome Browser: Enabling the Identification of Novel Serum Response Factor Target Genes</td>
<td>SRP042078</td>
<td align="right">26241044</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/26241044</td>
<td>Smooth muscle cells (colon)</td>
</tr>
<tr>
<td>(jejunum) Interstitial Cell of Cajal</td>
<td>Lee et al. PLoS One 2015</td>
<td>Smooth Muscle Cell Genome Browser: Enabling the Identification of Novel Serum Response Factor Target Genes</td>
<td>SRP042078</td>
<td align="right">26241044</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/26241044</td>
<td>Interstitial Cell of Cajal (jejunum)</td>
</tr>
<tr>
<td>(colon) Interstitial Cell of Cajal</td>
<td>Lee et al. PLoS One 2015</td>
<td>Smooth Muscle Cell Genome Browser: Enabling the Identification of Novel Serum Response Factor Target Genes</td>
<td>SRP042078</td>
<td align="right">26241044</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/26241044</td>
<td>Interstitial Cell of Cajal (colon)</td>
</tr>
<tr>
<td>(jejunum) GI PDGFRa+ cells</td>
<td>Lee et al. PLoS One 2015</td>
<td>Smooth Muscle Cell Genome Browser: Enabling the Identification of Novel Serum Response Factor Target Genes</td>
<td>SRP042078</td>
<td align="right">26241044</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/26241044</td>
<td>Gastrointestinal PDGFRa+ cells (jejunum)</td>
</tr>
<tr>
<td>(colon) GI PDGFRa+ cells</td>
<td>Lee et al. PLoS One 2015</td>
<td>Smooth Muscle Cell Genome Browser: Enabling the Identification of Novel Serum Response Factor Target Genes</td>
<td>SRP042078</td>
<td align="right">26241044</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/26241044</td>
<td>Gastrointestinal PDGFRa+ cells (colon)</td>
</tr>
<tr>
<td>(colon mucosa) GI PDGFRa+ cells</td>
<td>Lee et al. PLoS One 2015</td>
<td>Smooth Muscle Cell Genome Browser: Enabling the Identification of Novel Serum Response Factor Target Genes</td>
<td>SRP042078</td>
<td align="right">26241044</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/26241044</td>
<td>Gastrointestinal PDGFRa+ cells (colon mucosa)</td>
</tr>
<tr>
<td>Gut Mucosa</td>
<td>Lee et al. PLoS One 2015</td>
<td>Smooth Muscle Cell Genome Browser: Enabling the Identification of Novel Serum Response Factor Target Genes</td>
<td>SRP042078</td>
<td align="right">26241044</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/26241044</td>
<td>Gut Mucosa</td>
</tr>
<tr>
<td>(adult) Pancreatic alpha cells</td>
<td>Petrenko et al. Genes Dev. 2017</td>
<td>Pancreatic Î±- and Î²-cellular clocks have distinct molecular properties and impact on islet hormone secretion and gene expression</td>
<td>SRP100479</td>
<td align="right">28275001</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/28275001</td>
<td>Pancreatic alpha cells (7-16wks)</td>
</tr>
<tr>
<td>(adult) Pancreatic beta cells</td>
<td>Petrenko et al. Genes Dev. 2017</td>
<td>Pancreatic Î±- and Î²-cellular clocks have distinct molecular properties and impact on islet hormone secretion and gene expression</td>
<td>SRP100479</td>
<td align="right">28275001</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/28275001</td>
<td>Pancreatic beta cells (7-16wks)</td>
</tr>
<tr>
<td>(p0) Pancreatic beta cells</td>
<td>Qiu et al. Cell Metab. 2017</td>
<td>Deciphering Pancreatic Islet Î² Cell and Î± Cell Maturation Pathways and Characteristic Features at the Single-Cell Level</td>
<td>SRP090506</td>
<td align="right">28467935</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/28467935</td>
<td>Pancreatic beta cells (p0)</td>
</tr>
<tr>
<td>(p3) Pancreatic beta cells</td>
<td>Qiu et al. Cell Metab. 2017</td>
<td>Deciphering Pancreatic Islet Î² Cell and Î± Cell Maturation Pathways and Characteristic Features at the Single-Cell Level</td>
<td>SRP090506</td>
<td align="right">28467935</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/28467935</td>
<td>Pancreatic beta cells (p3)</td>
</tr>
<tr>
<td>(p9) Pancreatic beta cells</td>
<td>Qiu et al. Cell Metab. 2017</td>
<td>Deciphering Pancreatic Islet Î² Cell and Î± Cell Maturation Pathways and Characteristic Features at the Single-Cell Level</td>
<td>SRP090506</td>
<td align="right">28467935</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/28467935</td>
<td>Pancreatic beta cells (p9)</td>
</tr>
<tr>
<td>(p15) Pancreatic beta cells</td>
<td>Qiu et al. Cell Metab. 2017</td>
<td>Deciphering Pancreatic Islet Î² Cell and Î± Cell Maturation Pathways and Characteristic Features at the Single-Cell Level</td>
<td>SRP090506</td>
<td align="right">28467935</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/28467935</td>
<td>Pancreatic beta cells (p15)</td>
</tr>
<tr>
<td>(p18) Pancreatic beta cells</td>
<td>Qiu et al. Cell Metab. 2017</td>
<td>Deciphering Pancreatic Islet Î² Cell and Î± Cell Maturation Pathways and Characteristic Features at the Single-Cell Level</td>
<td>SRP090506</td>
<td align="right">28467935</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/28467935</td>
<td>Pancreatic beta cells (p18)</td>
</tr>
<tr>
<td>(p60) Pancreatic beta cells</td>
<td>Qiu et al. Cell Metab. 2017</td>
<td>Deciphering Pancreatic Islet Î² Cell and Î± Cell Maturation Pathways and Characteristic Features at the Single-Cell Level</td>
<td>SRP090506</td>
<td align="right">28467935</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/28467935</td>
<td>Pancreatic beta cells (p60)</td>
</tr>
<tr>
<td>(lung) Alveolar type 1</td>
<td>Yang et al. Development 2016</td>
<td>The development and plasticity of alveolar type 1 cells</td>
<td>SRP064621</td>
<td align="right">26586225</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/26586225</td>
<td>Alveolar type 1 (lung)</td>
</tr>
<tr>
<td>(kidney) Podocytes</td>
<td>Albert et al. J Mol Cell Biol. 2015</td>
<td>The GYF domain protein CD2BP2 is critical for embryogenesis and podocyte function</td>
<td>SRP034660</td>
<td align="right">26082520</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/26082520</td>
<td>Podocytes (kidney)</td>
</tr>
<tr>
<td>Hair follicle</td>
<td>Turkoz et al. J Invest Dermatol. 2016</td>
<td>The Notch Intracellular Domain Has an RBPj-Independent Role during Mouse Hair Follicular Development</td>
<td>SRP070433</td>
<td align="right">26940862</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/26940862</td>
<td>Hair follicle</td>
</tr>
<tr>
<td>Classical brown adipocyte</td>
<td>Shinoda et al. Nat Med. 2015</td>
<td>Genetic and functional characterization of clonally derived adult human brown adipocytes</td>
<td>ERP005997</td>
<td align="right">25774848</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/25774848</td>
<td>Classical brown adipocyte</td>
</tr>
<tr>
<td>Inguinal white adipocyte</td>
<td>Shinoda et al. Nat Med. 2015</td>
<td>Genetic and functional characterization of clonally derived adult human brown adipocytes</td>
<td>ERP005997</td>
<td align="right">25774848</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/25774848</td>
<td>Inguinal white adipocyte</td>
</tr>
<tr>
<td>Rosiglitazone beige adipocyte</td>
<td>Shinoda et al. Nat Med. 2015</td>
<td>Genetic and functional characterization of clonally derived adult human brown adipocytes</td>
<td>ERP005997</td>
<td align="right">25774848</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/25774848</td>
<td>Rosiglitazone-treated adipocyte (beige cell)</td>
</tr>
<tr>
<td>(bone) Mesenchymal cells</td>
<td>Patenaude and Perreault. J Immunol. 2016</td>
<td>Thymic Mesenchymal Cells Have a Distinct Transcriptomic Profile</td>
<td>SRP063829</td>
<td align="right">27183606</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/27183606</td>
<td>Mesenchymal cells (bone)</td>
</tr>
<tr>
<td>(skin) Mesenchymal cells</td>
<td>Patenaude and Perreault. J Immunol. 2016</td>
<td>Thymic Mesenchymal Cells Have a Distinct Transcriptomic Profile</td>
<td>SRP063829</td>
<td align="right">27183606</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/27183606</td>
<td>Mesenchymal cells (skin)</td>
</tr>
<tr>
<td>(thymus) Mesenchymal cells</td>
<td>Patenaude and Perreault. J Immunol. 2016</td>
<td>Thymic Mesenchymal Cells Have a Distinct Transcriptomic Profile</td>
<td>SRP063829</td>
<td align="right">27183606</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/27183606</td>
<td>Mesenchymal cells (thymus)</td>
</tr>
<tr>
<td>(testis) Sertoli cells</td>
<td>McClelland et al. Biol Reprod. 2015</td>
<td>Purification and Transcriptomic Analysis of Mouse Fetal Leydig Cells Reveals Candidate Genes for Specification of Gonadal Steroidogenic Cells</td>
<td>SRP053038</td>
<td align="right">25855264</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/25855264</td>
<td>Sertoli cells (testis)</td>
</tr>
<tr>
<td>(testis) Leydig cells</td>
<td>McClelland et al. Biol Reprod. 2015</td>
<td>Purification and Transcriptomic Analysis of Mouse Fetal Leydig Cells Reveals Candidate Genes for Specification of Gonadal Steroidogenic Cells</td>
<td>SRP053038</td>
<td align="right">25855264</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/25855264</td>
<td>Leydig cells (testis)</td>
</tr>
<tr>
<td>(testis) Interstitial cells</td>
<td>McClelland et al. Biol Reprod. 2015</td>
<td>Purification and Transcriptomic Analysis of Mouse Fetal Leydig Cells Reveals Candidate Genes for Specification of Gonadal Steroidogenic Cells</td>
<td>SRP053038</td>
<td align="right">25855264</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/25855264</td>
<td>Interstitial cells (testis)</td>
</tr>
<tr>
<td>(testis) Spermatogonial stem cells</td>
<td>Li et al. Oncotarget 2017</td>
<td>Systematic identification and comparison of expressed profiles of lncRNAs and circRNAs with associated co-expression and ceRNA networks in mouse germline stem cells</td>
<td>SRP091381</td>
<td align="right">28404936</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/28404936</td>
<td>Spermatogonial stem cells (testis)</td>
</tr>
<tr>
<td>(ovary) Female germline stem cells</td>
<td>Li et al. Oncotarget 2017</td>
<td>Systematic identification and comparison of expressed profiles of lncRNAs and circRNAs with associated co-expression and ceRNA networks in mouse germline stem cells</td>
<td>SRP091381</td>
<td align="right">28404936</td>
<td>https://www.ncbi.nlm.nih.gov/pubmed/28404936</td>
<td>Female germline stem cells (ovary)</td>
</tr>
</tbody></table>
